src/EPL-2.0.xml

```diff
@@ -0,0 +1,634 @@
+								the Program must also be made available as Source Code, in
+								accordance with section 3.2, and the Contributor must accompany
+								the Program with a statement that the Source Code for the
+								Program
+								is available under this Agreement, and informs Recipients
+								how to
+								obtain it in a reasonable manner on or through a medium
+								customarily
+								used for software exchange; and
+							</item>
+							<item>
+								<bullet>b)</bullet>
+								the Contributor may Distribute the Program under a license
+								different than this Agreement, provided that such license:
+								<list>
+									<item>
+										<bullet>i)</bullet>
+										effectively disclaims on behalf of all other Contributors all
+										warranties and conditions, express and implied, including
+										warranties
+										or conditions of title and non-infringement, and
+										implied
+										warranties
+										or conditions of merchantability and fitness
+										for a particular
+										purpose;
+									</item>
+									<item>
+										<bullet>ii)</bullet>
+										effectively excludes on behalf of all other Contributors all
+										liability for damages, including direct, indirect, special,
+										incidental
+										and consequential damages, such as lost profits;
+									</item>
+									<item>
+										<bullet>iii)</bullet>
+										does not attempt to limit or alter the recipients&#039; rights
+										in the
+										Source Code under section 3.2; and
+									</item>
+									<item>
+										<bullet>iv)</bullet>
+										requires any subsequent distribution of the Program by any
+										party
+										to be under a license that satisfies the requirements of
+										this
+										section 3.
+									</item>
+								</list>
+							</item>
+						</list>
+					</item>
+					<item>
+						<bullet>3.2</bullet>
+						When the Program is Distributed as Source Code:
+						<list>
+							<item>
+								<bullet>a)</bullet>
+								it must be made available under this Agreement, or if the
+								Program (i)
+								is combined with other material in a separate file or
+								files made
+								available
+								under a Secondary License, and (ii) the
+								initial Contributor attached
+								to
+								the Source Code the notice
+								described in Exhibit A of this
+								Agreement,
+								then the Program may be
+								made available under the terms of such
+								Secondary Licenses, and
+							</item>
+							<item>
+								<bullet>b)</bullet>
+								a copy of this Agreement must be included with each copy of the
+								Program.
+							</item>
+						</list>
+					</item>
+					<item>
+						<bullet>3.3</bullet>
+						Contributors may not remove or alter any copyright, patent,
+						trademark,
+						attribution notices, disclaimers of warranty, or
+						limitations of liability
+						("notices")
```

Comment:
I finally got around to comparing this with the [upstream text][1], and the only differences I see are curly/straight quotes (which I'm fine with ignoring) and then this line, which has double quotes here, but is using curly single quotes upstream:

```console
$ curl -s https://www.eclipse.org/legal/epl-2.0/ | grep '(.notices.)'
      (‘notices’) contained within the Program from any copy of the Program which
```

@waynebeaton, do you know if upstream intended to use single quotes there?  Or is the intention to use double (curly) quotes?

[1]: https://www.eclipse.org/legal/epl-2.0/

Reply:
As it was delivered to me, the text ony version uses straight double quotes, and the HTML version uses single curly quotes. That must have happened in translation. 

My vote is to ignore.

Reply:
curly v. straight quote doesn't matter for matching purposes: see 5.1.3 https://spdx.org/spdx-license-list/matching-guidelines

Reply:
> curly v. straight quote doesn't matter for matching purposes…

I agree.  I just wanted to make sure we represent the upstream preference for single vs. double (even though that doesn't impact matching, it's still nice to be consistent).  But as @waynebeaton points out, upstream uses double in their text/plain version and single in their text/html version, so (without a consistent upstream pattern to follow) it doesn't really matter which version we use here.